### PR TITLE
Draft: add sentry

### PR DESCRIPTION
**Pull request by arvryna**
* Fixes: https://github.com/requiemofthesouls/pigeomail/issues/17

### CODE REVIEW

File: monitoring/sentry.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import (
+	"log"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/spf13/viper"
+)
+
+func InitSentry() {
+
+	// TODO: Need help in unmarshalling this config properly
```
Comment (arvryna) [inline monitoring/sentry.go:12]: @requiemofthesouls : Need your help here

Comment (requiemofthesouls) [reply monitoring/sentry.go:12]: lets make a call on Saturday then

**Review by requiemofthesouls — CHANGES_REQUESTED** [review]

**Review by requiemofthesouls — CHANGES_REQUESTED** [review]
need to pull actual master

**Comment by arvryna** [comment]
this PR will be closed, will raise a different PR today

**Pull request closed without merging**